### ISSUE TITLE
2.9.0 Stable Release

### ISSUE THREAD

**Issue by danahertzberg** [az-digital/az_quickstart#2975]
- #2972 
- #2973 
- #2974

**Comment by joeparsons** [comment]
## Release Notes snippet

```
## New changes since 2.9.0-beta2

## Important notes
- **Important**: Updating from a Quickstart codebase less than 2.6.x is no longer supported.
- To update to a Quickstart 2.9.x release (e.g. 2.9.0) from a version less than 2.6.8 you must first update to 2.6.8 or greater and run database updates. Database updates added to Quickstart prior to version 2.6.0 have been removed in 2.7.x and 2.8.x and some core modules included in Drupal 9 have been removed in Drupal 10.
- **Quickstart 2.7.x will longer receive security support. Sites on Quickstart 2.7.x or earlier should upgrade to a supported release as soon as possible.**. See our [release policy](https://github.com/az-digital/az_quickstart/blob/main/RELEASES.md#release-overview) for more information.
- For more relevant info about upgrading from 2.6.x see [Quickstart 2.7.0 Upgrade Notes](https://github.com/az-digital/az_quickstart/wiki/Quickstart-2.7.0-Upgrade-Notes).

## Major changes (not included in 2.8.x)
- Upgrade Drupal Core to 10.2.x See https://www.drupal.org/project/drupal/releases/10.2.0

### Changes to site-owner-managed files (Pantheon Quickstart 2 upstream users can safely ignore this.)

### Deprecated modules slated for removal
- [Block Content Permissions](https://www.drupal.org/project/block_content_permissions)
- [Media Library Theme Reset](https://www.drupal.org/project/media_library_theme_reset) (No longer supported by maintainer)

## All changes
```

**Comment by joeparsons** [comment]
# Draft release notes snippet
`git log 2.9.0-beta2..HEAD --pretty='format:- %s' `


## New changes since 2.9.0-beta2
- Closes #3113 Standard text format cannot be edited in the UI without removing allowed tags (#3114)
- Back to dev.

## Important notes

- **Important**: Sites upgrading from an earlier version of Quickstart should apply database updates and uninstall the [Block Content Permissions](https://www.drupal.org/project/block_content_permissions) after upgrading because the additional permissions provided by that module are now available in Drupal core (as of Drupal 10.1.0).  It may also be necessary to use the `drush az-core-config-add-permissions` command to ensure the proper block permissions are assigned to Quickstart provided permissions after upgrading Quickstart and uninstalling the Block Content Permissions module.
- **Important**: Updating from a Quickstart codebase less than 2.6.x is no longer supported.
- To update to a Quickstart 2.9.x release (e.g. 2.9.0) from a version less than 2.6.8 you must first update to 2.6.8 or greater and run database updates. Database updates added to Quickstart prior to version 2.6.0 have been removed in 2.7.x and 2.8.x and some core modules included in Drupal 9 have been removed in Drupal 10.
- **Quickstart 2.7.x will longer receive security support. Sites on Quickstart 2.7.x or earlier should upgrade to a supported release as soon as possible.**. See our [release policy](https://github.com/az-digital/az_quickstart/blob/main/RELEASES.md#release-overview) for more information.
- For more relevant info about upgrading from 2.6.x see [Quickstart 2.7.0 Upgrade Notes](https://github.com/az-digital/az_quickstart/wiki/Quickstart-2.7.0-Upgrade-Notes).

## Major changes (not included in 2.8.x)
- Upgrade Drupal Core to 10.2.x See https://www.drupal.org/project/drupal/releases/10.2.0

### Deprecated modules slated for removal
- [Block Content Permissions](https://www.drupal.org/project/block_content_permissions)
- [Media Library Theme Reset](https://www.drupal.org/project/media_library_theme_reset) (No longer supported by maintainer)

## All changes

**Comment by trackleft** [comment]
Updated 
```

## New changes since 2.9.0-beta2
- Closes #3113 Standard text format cannot be edited in the UI without removing allowed tags (#3114)

## Important notes
- **Important**: [Database updates](https://github.com/az-digital/az_quickstart/wiki/Maintaining-Quickstart-2-Sites#applying-database-updates) and [Distribution configuration updates](https://github.com/az-digital/az_quickstart/wiki/Maintaining-Quickstart-2-Sites#applying-distribution-configuration-updates) should _always_ be applied when updating Quickstart sites.
- **Important**: After applying database and configuration updates, sites upgrading from an earlier minor version of Quickstart to 2.9.x should also uninstall the [Block Content Permissions](https://www.drupal.org/project/block_content_permissions) module after upgrading because many of the permissions provided by that module are now available in Drupal core (as of Drupal 10.1.0).  It may also be necessary to use the `drush az-core-config-add-permissions` command to ensure the proper block permissions are assigned to Quickstart provided permissions after upgrading Quickstart and uninstalling the Block Content Permissions module.
- **Important**: Updating from a Quickstart codebase less than 2.6.x is no longer supported.
- To update to a Quickstart 2.9.x release (e.g. 2.9.0) from a version less than 2.6.8 you must first update to 2.6.8 or greater and run database updates. Database updates added to Quickstart prior to version 2.6.0 have been removed in 2.7.x and 2.8.x and some core modules included in Drupal 9 have been removed in Drupal 10.
- **For Site Updaters Managing Their Own Environments**: With the inclusion of a minor Drupal update in this release, it's imperative for site updaters who manage their own environments to thoroughly review the Drupal release notes and their own setups.   Please review https://github.com/az-digital/az-quickstart-scaffolding for the latest recommendations.
- **Quickstart 2.7.x will no longer receive security support. Sites on Quickstart 2.7.x or earlier should upgrade to a supported release as soon as possible**. See our [release policy](https://github.com/az-digital/az_quickstart/blob/main/RELEASES.md#release-overview) for more information.
- For more relevant info about upgrading from 2.6.x see [Quickstart 2.7.0 Upgrade Notes](https://github.com/az-digital/az_quickstart/wiki/Quickstart-2.7.0-Upgrade-Notes).

## Major changes (not included in 2.8.x)
- Upgrade Drupal Core to 10.2.x See https://www.drupal.org/project/drupal/releases/10.2.0
- #3099 Update Drupal core to 10.2.2 for SA-CORE-2024-001 (#3100)

### Deprecated modules slated for removal
- [Block Content Permissions](https://www.drupal.org/project/block_content_permissions) (No longer needed as of Drupal 10.1.0)
- [Media Library Theme Reset](https://www.drupal.org/project/media_library_theme_reset) (No longer supported by maintainer)

## All changes not included in 2.8.x

### Bugfix
- Closes #3113 Standard text format cannot be edited in the UI without removing allowed tags (#3114)
- Closes #3102 Some CKEditor editable tags are removed and can't be restored (#3107)
- Closes #3097 Changes to rendered HTML after update to Drupal 10.2 (#3106)
- Closes #3084 Event edit page displays User deprecated function error (#3085)
- Closes #3082 Vertical tabs in Media Library modal are missing styling (#3083)
- Closes #3063 Fix permissions for reverting block content (#3064)
- Closes #2757: Fix typo in README.md (#3061)
- Closes #3014, Closes #3017: Update roles with additional block content permissions added to Drupal core 10.1 and deprecate block_content_permissions module. (#3027)
- Closes #2967, Closes #1138, Closes #3025: Upgrade Drupal core to 10.2.x. (#2968)
- Closes #2911: PHP 8.2 compatibility. (#2915)
- Close #2962 handle empty values in TextFormatRecognizer.php (#2963)

### Breaking or Visual Change to Existing Behavior
- Close #3028 Update throbber template to use Arizona Bootstrap spinner. (#3029)

### Drupal Contrib Project Updates

- Update drupal/ckeditor_bs_grid requirement from 2.0.10 to 2.0.11 (#3094)
- Update drupal/migrate_plus requirement from 6.0.1 to 6.0.2 (#3093)
- Update drupal/paragraphs requirement from 1.16.0 to 1.17.0 (#3095)
- Update drupal/quick_node_clone requirement from 1.16.0 to 1.17.0 (#3088)
- Update drupal/core-recommended requirement from 10.2.0 to 10.2.1 (#3062)
- Update drupal/date_ap_style requirement from 1.9.0 to 2.0.0 (#3054)
- Update citation-style-language/styles requirement from 0.0.330 to 0.0.343 (#3055)
- Update citation-style-language/styles requirement from 0.0.325 to 0.0.330 (#2970)
- Update drupal/blazy requirement from 2.21.0 to 2.22.0 (#2914)
- Update drupal/date_ap_style requirement from 1.7.0 to 1.9.0 (#2921)
- Update drupal/inline_entity_form requirement from 3.0.0-rc15 to 3.0.0-rc17 (#3036)
- Update drupal/webform requirement from 6.2.0 to 6.2.2 (#3005)
- Closes #2938: Update drupal/inline_entity_form requirement from 2.0.0-rc9 to 3.0.0-rc15 (#3007)
- Update drupal/media_entity_file_replace requirement from 1.1 to 1.2.0 (#2954)

### Editor Experience

- Closes #2927 Update photo field label on news add/edit form to 'Featured Photo' to match what it is called on field_az_featured_image_as_thumb (#2935)
- Close #2930 Add suggested character limit of 160 to all az_summary fields. (#2937)
- Close #2928 Always show summary fields when adding or editing content. (#2932)

### Developer Experience

- Close #3022 Change argument array syntax in .service.yml files to make it easier to review changes. (#3023)
- Update PHPUnit config file using phpunit --migrate-configuration (#2788)
- Closes #2905 Clean up various PHPCS warnings (#2906)
- #2967: Coding standards fixes for Drupal 10.2.x (#2978)
- Closes #2197 use card title field in demo content, Closes #2426 Add view paragraphs to demo content. (#2422)

### Other

- Back to dev.
- Preparing to tag 2.9.0-beta2.
- Back to dev.
- Preparing to tag 2.9.0-beta1.
- Back to dev.
- Preparing to tag 2.9.0-alpha1.
- Prepare repository for new minor release branch 2.9.x (#3051)
- Prepare for new minor release 2.9.x


## Changes also included in 2.8.x

### Bugfix

- Close #3104 fix typo in az_news_feeds schema. (#3105)
- Closes #2758: Add patch to address Easy Breadcrumb deprecation warnings. (#2759)
- Closes #3077: Address PHP 8.1+ deprecation warnings caused by card links with empty URLs. (#3078)
- #2084: Add DB update to uninstall now unsupported media_library_theme_reset module. (#3040)
- #3014: Add critical new block_content permissions to AZQS roles (added in Drupal 10.1.x) (#3038)
- Closes #2952 Clone permissions in the Quick Node Clone module do not reflect custom access rules (#2953)
- Closes #2852 Deprecation warnings while news feed is importing (#2969)
- Closes #2916 Exclude BigPipe module from Quickstart installs on Pantheon (#2917)
- Closes #2736: Remove whitespace on cards with no body or link. (#2940)
- Closes #2102 Error upon save when linking to a document in a Card Link URL field (#2900)
- Fixes #2888 Marketing Cloud Event Specificity (#2901)
- Closes #2944 Update style.css text field widths (#2945)
- Closes #2169 Close icon broken in media library edit module's modal (#2956)
- Closes #2946 Video Thumbnail in Page/Event/News Editor Too Small (#2949)

### Breaking or Visual Change to Existing Behavior
- Closes #3065: Remove text decoration from links in az_publication.module (#3066)
- Close #2800 Update Global Footer with new X/Twitter Icon. (#3056)

### Drupal Contrib Project Updates

- Update drupal/core-recommended requirement from 10.1.6 to 10.1.7 (#3000)
- Closes #3008: Update seboettg/citeproc-php requirement from 2.6.0 to 2.6.2 (#3009)
- Update citation-style-language/styles requirement from 0.0.303 to 0.0.325 (#2942)
- Update drupal/viewsreference requirement from 2.0.0-beta6 to 2.0.0-beta7 (#2895)
- Update drupal/core-recommended requirement from 10.1.5 to 10.1.6 (#2894)
- Update drupal/entity_embed requirement from 1.4.0 to 1.5.0 (#2910)

### Editor Experience

- Close #2933 Update external link field help text to mention University calendar, and to be more concise. (#2934)
- Closes #1675 Minor update to help text for "Where" section of Event Page (#2977)

### Developer Experience

- #2084 Add media_library_theme_reset to list of deprecated Quickstart modules. (#2996)
- Closes #1946 Embedded images in demo content are not found (#2995)
- Closes #2702: Add unpublished content layout to Marketing Cloud (#2782)
- Closes #2907 Make Manual Migration Lookup DB configurable. (#2908)
- Closes #2918: Add AZ Bootstrap library dependency to node form overrides library. (#2919)
- Closes #2671: Enable copy options based on content type (#2880)
```
```

**Comment by trackleft** [comment]
```

Updated 
```

## New changes since 2.9.0-beta2
- Closes #3113 Standard text format cannot be edited in the UI without removing allowed tags (#3114)

## Important notes
- **Important**: [Database updates](https://github.com/az-digital/az_quickstart/wiki/Maintaining-Quickstart-2-Sites#applying-database-updates) and [Distribution configuration updates](https://github.com/az-digital/az_quickstart/wiki/Maintaining-Quickstart-2-Sites#applying-distribution-configuration-updates) should _always_ be applied when updating Quickstart sites.
- **Important**: After applying database and configuration updates, sites upgrading from an earlier minor version of Quickstart to 2.9.x should also uninstall the [Block Content Permissions](https://www.drupal.org/project/block_content_permissions) module after upgrading because many of the permissions provided by that module are now available in Drupal core (as of Drupal 10.1.0).  It may also be necessary to use the `drush az-core-config-add-permissions` command to ensure the proper block permissions are assigned to Quickstart provided permissions after upgrading Quickstart and uninstalling the Block Content Permissions module.
- **Important**: Updating from a Quickstart codebase less than 2.6.x is no longer supported.
- To update to a Quickstart 2.9.x release (e.g. 2.9.0) from a version less than 2.6.8 you must first update to 2.6.8 or greater and run database updates. Database updates added to Quickstart prior to version 2.6.0 have been removed in 2.7.x and 2.8.x and some core modules included in Drupal 9 have been removed in Drupal 10.
- **For Site Updaters Managing Their Own Environments**: With the inclusion of a minor Drupal update in this release, it's imperative for site updaters who manage their own environments to thoroughly review the Drupal release notes and their own setups.   Please review https://github.com/az-digital/az-quickstart-scaffolding for the latest recommendations.
- **Quickstart 2.7.x will no longer receive security support. Sites on Quickstart 2.7.x or earlier should upgrade to a supported release as soon as possible**. See our [release policy](https://github.com/az-digital/az_quickstart/blob/main/RELEASES.md#release-overview) for more information.
- For more relevant info about upgrading from 2.6.x see [Quickstart 2.7.0 Upgrade Notes](https://github.com/az-digital/az_quickstart/wiki/Quickstart-2.7.0-Upgrade-Notes).

## Major changes (not included in 2.8.x)
- Upgrade Drupal Core to 10.2.x See https://www.drupal.org/project/drupal/releases/10.2.0
- #3099 Update Drupal core to 10.2.2 for SA-CORE-2024-001 (#3100)

### Deprecated modules slated for removal
- [Block Content Permissions](https://www.drupal.org/project/block_content_permissions) (No longer needed as of Drupal 10.1.0)
- [Media Library Theme Reset](https://www.drupal.org/project/media_library_theme_reset) (No longer supported by maintainer)

## All changes not included in 2.8.x

### Bugfix
- Closes #3113 Standard text format cannot be edited in the UI without removing allowed tags (#3114)
- Closes #3102 Some CKEditor editable tags are removed and can't be restored (#3107)
- Closes #3097 Changes to rendered HTML after update to Drupal 10.2 (#3106)
- Closes #3084 Event edit page displays User deprecated function error (#3085)
- Closes #3082 Vertical tabs in Media Library modal are missing styling (#3083)
- Closes #3063 Fix permissions for reverting block content (#3064)
- Closes #2757: Fix typo in README.md (#3061)
- Closes #3014, Closes #3017: Update roles with additional block content permissions added to Drupal core 10.1 and deprecate block_content_permissions module. (#3027)
- Closes #2967, Closes #1138, Closes #3025: Upgrade Drupal core to 10.2.x. (#2968)
- Closes #2911: PHP 8.2 compatibility. (#2915)
- Close #2962 handle empty values in TextFormatRecognizer.php (#2963)

### Breaking or Visual Change to Existing Behavior
- Close #3028 Update throbber template to use Arizona Bootstrap spinner. (#3029)

### Drupal Contrib Project Updates

- Update drupal/ckeditor_bs_grid requirement from 2.0.10 to 2.0.11 (#3094)
- Update drupal/migrate_plus requirement from 6.0.1 to 6.0.2 (#3093)
- Update drupal/paragraphs requirement from 1.16.0 to 1.17.0 (#3095)
- Update drupal/quick_node_clone requirement from 1.16.0 to 1.17.0 (#3088)
- Update drupal/core-recommended requirement from 10.2.0 to 10.2.1 (#3062)
- Update drupal/date_ap_style requirement from 1.9.0 to 2.0.0 (#3054)
- Update citation-style-language/styles requirement from 0.0.330 to 0.0.343 (#3055)
- Update citation-style-language/styles requirement from 0.0.325 to 0.0.330 (#2970)
- Update drupal/blazy requirement from 2.21.0 to 2.22.0 (#2914)
- Update drupal/date_ap_style requirement from 1.7.0 to 1.9.0 (#2921)
- Update drupal/inline_entity_form requirement from 3.0.0-rc15 to 3.0.0-rc17 (#3036)
- Update drupal/webform requirement from 6.2.0 to 6.2.2 (#3005)
- Closes #2938: Update drupal/inline_entity_form requirement from 2.0.0-rc9 to 3.0.0-rc15 (#3007)
- Update drupal/media_entity_file_replace requirement from 1.1 to 1.2.0 (#2954)

### Editor Experience

- Closes #2927 Update photo field label on news add/edit form to 'Featured Photo' to match what it is called on field_az_featured_image_as_thumb (#2935)
- Close #2930 Add suggested character limit of 160 to all az_summary fields. (#2937)
- Close #2928 Always show summary fields when adding or editing content. (#2932)

### Developer Experience

- Close #3022 Change argument array syntax in .service.yml files to make it easier to review changes. (#3023)
- Update PHPUnit config file using phpunit --migrate-configuration (#2788)
- Closes #2905 Clean up various PHPCS warnings (#2906)
- #2967: Coding standards fixes for Drupal 10.2.x (#2978)
- Closes #2197 use card title field in demo content, Closes #2426 Add view paragraphs to demo content. (#2422)

### Other

- Back to dev.
- Preparing to tag 2.9.0-beta2.
- Back to dev.
- Preparing to tag 2.9.0-beta1.
- Back to dev.
- Preparing to tag 2.9.0-alpha1.
- Prepare repository for new minor release branch 2.9.x (#3051)
- Prepare for new minor release 2.9.x


## Changes also included in 2.8.x

### Bugfix

- Close #3104 fix typo in az_news_feeds schema. (#3105)
- Closes #2758: Add patch to address Easy Breadcrumb deprecation warnings. (#2759)
- Closes #3077: Address PHP 8.1+ deprecation warnings caused by card links with empty URLs. (#3078)
- #2084: Add DB update to uninstall now unsupported media_library_theme_reset module. (#3040)
- #3014: Add critical new block_content permissions to AZQS roles (added in Drupal 10.1.x) (#3038)
- Closes #2952 Clone permissions in the Quick Node Clone module do not reflect custom access rules (#2953)
- Closes #2852 Deprecation warnings while news feed is importing (#2969)
- Closes #2916 Exclude BigPipe module from Quickstart installs on Pantheon (#2917)
- Closes #2736: Remove whitespace on cards with no body or link. (#2940)
- Closes #2102 Error upon save when linking to a document in a Card Link URL field (#2900)
- Fixes #2888 Marketing Cloud Event Specificity (#2901)
- Closes #2944 Update style.css text field widths (#2945)
- Closes #2169 Close icon broken in media library edit module's modal (#2956)
- Closes #2946 Video Thumbnail in Page/Event/News Editor Too Small (#2949)
- Closes #2918: Add AZ Bootstrap library dependency to node form overrides library. (#2919)

### Breaking or Visual Change to Existing Behavior
- Closes #3065: Remove text decoration from links in az_publication.module (#3066)
- Close #2800 Update Global Footer with new X/Twitter Icon. (#3056)

### Drupal Contrib Project Updates

- Update drupal/core-recommended requirement from 10.1.6 to 10.1.7 (#3000)
- Closes #3008: Update seboettg/citeproc-php requirement from 2.6.0 to 2.6.2 (#3009)
- Update citation-style-language/styles requirement from 0.0.303 to 0.0.325 (#2942)
- Update drupal/viewsreference requirement from 2.0.0-beta6 to 2.0.0-beta7 (#2895)
- Update drupal/core-recommended requirement from 10.1.5 to 10.1.6 (#2894)
- Update drupal/entity_embed requirement from 1.4.0 to 1.5.0 (#2910)

### Editor Experience

- Close #2933 Update external link field help text to mention University calendar, and to be more concise. (#2934)
- Closes #1675 Minor update to help text for "Where" section of Event Page (#2977)
- Closes #2671: Enable copy options based on content type (#2880)
- Closes #2702: Add unpublished content layout to Marketing Cloud (#2782)

### Developer Experience

- #2084 Add media_library_theme_reset to list of deprecated Quickstart modules. (#2996)
- Closes #1946 Embedded images in demo content are not found (#2995)
- Closes #2907 Make Manual Migration Lookup DB configurable. (#2908)
```

**Comment by joeparsons** [comment]
https://github.com/az-digital/az_quickstart/releases/tag/2.9.0
